src/ProofOfConcepts/DrawerDescList/ConnectorDrawer.tsx

```diff
@@ -0,0 +1,71 @@
+import { ConnectorStatus } from '@app/components/ConnectorStatus/ConnectorStatus';
+import React, { FunctionComponent, ReactNode } from 'react';
+
+import {
+  Drawer,
+  DrawerContent,
+  DrawerPanelContent,
+} from '@patternfly/react-core';
+
+import { Connector } from '@rhoas/connector-management-sdk';
+
+import './ConnectorDrawer.css';
+import { ConnectorDrawerData } from './ConnectorDrawerData';
+import { DrawerHeader } from './DrawerHeader';
+
+export type ConnectorDrawerProps = {
+  connector: Connector | undefined;
+  children: ReactNode;
+  actionsMenu: ReactNode;
+  onClose: () => void;
+  onDuplicateConnector: (id: string) => void;
+};
+
+export const ConnectorDrawer: FunctionComponent<ConnectorDrawerProps> = ({
+  connector,
+  actionsMenu,
+  children,
+  onClose,
+  onDuplicateConnector,
+}) => {
+  return (
+    <Drawer isExpanded={typeof connector !== 'undefined'}>
+      <DrawerContent
+        panelContent={
+          typeof connector !== 'undefined' && (
+            <DrawerPanelContent widths={{ default: 'width_50' }}>
+              <DrawerHeader
+                drawerHeading={connector.name}
+                status={
+                  <ConnectorStatus
+                    desiredState={connector.desired_state}
+                    name={connector.name}
+                    state={connector.status?.state!}
+                  />
+                }
+                actionsMenu={actionsMenu}
+                onClose={onClose}
+              />
+              <ConnectorDrawerData
```

Comment:
Just to further expand on [my comment](https://github.com/redhat-developer/app-services-ui-components/pull/617#issuecomment-1424132325), it's good to have all the extra stuff to visualize the final thing, but I'd suggest probably just doing one mock-up or component that's used in the Story and share that amidst all of the variants so that the changes can focus more on the message statistics display.